cmd/tailscaled/depaware.txt

```diff
@@ -218,9 +238,104 @@ tailscale.com/cmd/tailscaled dependencies: (generated by github.com/tailscale/de
+   L    k8s.io/apimachinery/pkg/runtime/serializer/protobuf          from k8s.io/apimachinery/pkg/runtime/serializer
+   L    k8s.io/apimachinery/pkg/runtime/serializer/recognizer        from k8s.io/apimachinery/pkg/runtime/serializer+
+   L    k8s.io/apimachinery/pkg/runtime/serializer/versioning        from k8s.io/apimachinery/pkg/runtime/serializer
+   L    k8s.io/apimachinery/pkg/selection                            from k8s.io/apimachinery/pkg/apis/meta/v1+
+   L    k8s.io/apimachinery/pkg/types                                from k8s.io/api/authentication/v1+
+   L    k8s.io/apimachinery/pkg/util/errors                          from k8s.io/apimachinery/pkg/runtime+
+   L    k8s.io/apimachinery/pkg/util/framer                          from k8s.io/apimachinery/pkg/runtime/serializer/json+
+   L    k8s.io/apimachinery/pkg/util/intstr                          from k8s.io/api/apps/v1+
+   L    k8s.io/apimachinery/pkg/util/json                            from k8s.io/apimachinery/pkg/apis/meta/v1/unstructured+
+   L    k8s.io/apimachinery/pkg/util/naming                          from k8s.io/apimachinery/pkg/runtime
+   L    k8s.io/apimachinery/pkg/util/net                             from k8s.io/apimachinery/pkg/watch
+   L    k8s.io/apimachinery/pkg/util/runtime                         from k8s.io/apimachinery/pkg/apis/meta/v1+
+   L    k8s.io/apimachinery/pkg/util/sets                            from k8s.io/apimachinery/pkg/labels+
+   L    k8s.io/apimachinery/pkg/util/validation                      from k8s.io/apimachinery/pkg/labels
+   L    k8s.io/apimachinery/pkg/util/validation/field                from k8s.io/apimachinery/pkg/labels+
+   L    k8s.io/apimachinery/pkg/util/yaml                            from k8s.io/apimachinery/pkg/runtime/serializer/json
+   L    k8s.io/apimachinery/pkg/watch                                from k8s.io/apimachinery/pkg/apis/meta/v1
+   L    k8s.io/apimachinery/third_party/forked/golang/reflect        from k8s.io/apimachinery/pkg/conversion
+   L    k8s.io/client-go/kubernetes/scheme                           from tailscale.com/kube/apis/v1alpha1
+   L    k8s.io/klog/v2                                               from k8s.io/apimachinery/pkg/apis/meta/v1/unstructured+
+   L    k8s.io/klog/v2/internal/buffer                               from k8s.io/klog/v2
+   L    k8s.io/klog/v2/internal/clock                                from k8s.io/klog/v2
+   L    k8s.io/klog/v2/internal/dbg                                  from k8s.io/klog/v2
+   L    k8s.io/klog/v2/internal/serialize                            from k8s.io/klog/v2
+   L    k8s.io/klog/v2/internal/severity                             from k8s.io/klog/v2+
+   L    k8s.io/utils/clock                                           from tailscale.com/kube
+   L    k8s.io/utils/internal/third_party/forked/golang/net          from k8s.io/utils/net
+   L    k8s.io/utils/net                                             from k8s.io/apimachinery/pkg/util/net+
```

Comment:
previously k8s deps substantially increased the `tailscaled` binary size, do you know if this has similar impact?